### ISSUE TITLE
Permission issue starting DIYLC in Mac OS 15.1.1 Sequoia

### ISSUE THREAD

**Issue by mramonlopez** [bancika/diy-layout-creator#850]
If you install the last version of DITLC on a Mac with OS 15 Sequoia and try to run it, you could get the error 

> The application "Finder" does
> not have permission to open
> "(null)".

It's a problem with the signing of the .app file. You can fix it in your computer running:

`codesign --remove-signature /Applications/DIYLC.app
`

and then:

`codesign --force --deep --sign - /Applications/DIYLC.app
`

Source: https://forums.macrumors.com/threads/finder-does-not-have-permission-to-open-null.2441374/?post=33535657#post-33535657

**Comment by LudfisterSound** [comment]
This worked for me to get rid of the dock null errors but I did still have to attempt to open the app and then go to Privacy & Security in Setting and allow the app to run under the Security section all the way at the bottom.

**Comment by bancika** [comment]
Yes, that's due to Apple's requirement for everyone to distribute apps
through apple store. I plan to offer the app there but it will probably
cost a few dollars to download it there because it will cost me some cash
for apple developer account and it will require more effort to publish
there too. Stay tuned, I plan to do it soon

**Comment by MarsManiac** [comment]
Also an issue for me.  PowerBook, Intel, OSX 15.2.  Thanks for prompt attention.   NRN

**Comment by mbvanemby** [comment]
The solution above worked well for me. Thanks!

**Comment by MarsManiac** [comment]
Those commands worked without error.  Then I get 

'You can’t open the application “DIYLC.app” because it may be damaged or incomplete.'

Is there a workaround for this?

**Comment by bancika** [comment]
You must enable it as a 3rd party app, link here
https://github.com/bancika/diy-layout-creator
